Week1/homework/app.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+const root = document.getElementById("root");
+
+const hyfGithubRepositories = `https://api.github.com/orgs/HackYourFuture/repos?per_page=100`;
+
+
+function fetchJSON(callback, url) {
+    let xhr = new XMLHttpRequest();
+    xhr.open("GET", url, true);
+    xhr.onreadystatechange = () => {
+        if (xhr.readyState === 4 && xhr.status === 200) {
+            let response = JSON.parse(xhr.response);
+            callback(response);
+            return response;
+
+        } else if (xhr.status === 404) {
+            console.log("Error: resource not found");
```

Comment:
The assignment says that you have to display the error on the screen, printing it to console isn't effective enough as people don't usually check the console log.